react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -84,6 +93,7 @@ class Thumbnail extends Component<Props> {
 
         // Bind event handlers so they are only bound once for every instance.
         this._onClick = this._onClick.bind(this);
+        this._onShowRemoteVideoMenu = this._onShowRemoteVideoMenu.bind(this);
```

Comment:
I'm commenting here because it's the first occurence of the term "remove video menu". There is nothing in this menu which is related with the video. This is a "remote participant menu" if you will.